app/_hub/kong-inc/request-validator/overview/_index.md

```diff
@@ -419,13 +343,17 @@ In this example, use the plugin to validate a request's path parameter.
 
 4. In these step examples, validation ensures that `status_code` is a number and the body contains a parameter called `name`.
 
-   A proxy request with a non-numerical status code is blocked:
+   For example, if you try to send a request to an existing route with the path `/status/abc`:
```

Comment:
```suggestion
   For example, if you try to send a request to an existing route with the path `/status/123`:
```

Reply:
Ah - this one is right, the example is wrong. I'll fix that.